ncm-metaconfig/src/main/pan/components/metaconfig/config-common.pan

```diff
@@ -1,17 +0,0 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
-
-unique template components/${project.artifactId}/config-common;
-
-include { 'components/${project.artifactId}/schema' };
-
-# Set prefix to root of component configuration.
-prefix '/software/components/${project.artifactId}';
-
-#'version' = '${project.version}';
-#'package' = 'NCM::Component';
```

Comment:
> i always tought that there was a mechanism somewhere that also triggered the component if the > component package was updated, but i wouldn't know how that worked, so thinking about it, that > probably doesn't happen :smile: (well, maybe :confused: ) 

It definitely does on the version we are using because I managed to trigger a run of all dispatched components across our plant when I changed the schema of ncm-spma a year or so back. One of machines was misconfigured and so when chkconfig ran it disabled a service which had startstop=true and I had to explain to people why the schema change to /software/packages had shutdown their (unregistered) database service.

OTOH I don't know which version of ncm-cdispd that was so perhaps a subsequent refactor removed that?